planner/planner.go

```diff
@@ -498,6 +506,25 @@ func (p *MyPlanner) SetPlanSavedStatusForUser(ctx *gin.Context, numResults int,
 	return nil
 }
 
+func (p *MyPlanner) queuePlanningJobsForAllPriceLevels(req *PlanningRequest) error {
+	curTime := time.Now()
+	for l := range []POI.PriceLevel{POI.PriceLevelZero, POI.PriceLevelOne, POI.PriceLevelTwo, POI.PriceLevelThree, POI.PriceLevelFour} {
+		newReq := *req
+		newReq.PriceLevel = POI.PriceLevel(l)
+
```

Comment:
what is this portion of the code doing ? Lets discuss over the meeting

Reply:
It queues jobs for all price levels given other planning specs intact